python/lsst/ts/scheduler/utils/fbs_utils.py

```diff
@@ -0,0 +1,167 @@
+            "fieldDec": "dec",
+            "altitude": "alt",
+            "azimuth": "az",
+            "filter": "filter",
+            "airmass": "airmass",
+            "skyBrightness": "skybrightness",
+            "cloud": "clouds",
+            "seeingFwhm500": "FWHM_500",
+            "seeingFwhmGeom": "FWHM_geometric",
+            "seeingFwhmEff": "FWHMeff",
+            "fiveSigmaDepth": "fivesigmadepth",
+            "slewTime": "slewtime",
+            "slewDistance": "slewdist",
+            "paraAngle": "pa",
+            "rotTelPos": "rotTelPos",
+            "rotTelPos_backup": "rotTelPos_backup",
+            "rotSkyPos": "rotSkyPos",
+            "rotSkyPos_desired": "rotSkyPos_desired",
+            "moonRA": "moonRA",
+            "moonDec": "moonDec",
+            "moonAlt": "moonAlt",
+            "moonAz": "moonAz",
+            "moonDistance": "moonDist",
+            "moonPhase": "moonPhase",
+            "sunAlt": "sunAlt",
+            "sunAz": "sunAz",
+            "solarElong": "solarElong",
+            "note": "note",
+        }
+        # Column(s) not bothering to remap:  'observationStartTime': None,
+        self.inv_map = {v: k for k, v in self.convert_dict.items()}
+        # angles to convert
+        self.angles_rad2deg = [
+            "fieldRA",
+            "fieldDec",
+            "altitude",
+            "azimuth",
+            "slewDistance",
+            "paraAngle",
+            "rotTelPos",
+            "rotSkyPos",
+            "rotSkyPos_desired",
+            "rotTelPos_backup",
+            "moonRA",
+            "moonDec",
+            "moonAlt",
+            "moonAz",
+            "moonDistance",
+            "sunAlt",
+            "sunAz",
+            "sunRA",
+            "sunDec",
+            "solarElong",
+            "cummTelAz",
+        ]
+        # Put LMST into degrees too
+        self.angles_hours2deg = ["observationStartLST"]
+
+    def obs2opsim(
+        self,
+        obs_array: np.ndarray,
+        filename: str,
+    ) -> None:
+        """Convert an array of observations into a pandas dataframe
+        with Opsim schema.
+
+        Parameters
+        ----------
+        obs_array : np.ndarray
+            Array of observations.
+        filename : str
+            Name of the database file.
+        """
+
+        df = pd.DataFrame(obs_array)
+        df = df.rename(index=str, columns=self.inv_map)
+        for colname in self.angles_rad2deg:
+            df[colname] = np.degrees(df[colname])
+        for colname in self.angles_hours2deg:
+            df[colname] = df[colname] * 360.0 / 24.0
+
+        if filename is not None:
+            con = sqlite3.connect(filename)
+            df.to_sql(
+                "observations",
+                con,
+                if_exists="append",
+                index=False,
+            )
+
+    def opsim2obs(self, filename: str) -> np.ndarray:
+        """convert an opsim schema dataframe into an observation array.
```

Comment:
`convert` -> `Convert`. Also please describe that the Opsim dataframe gets loaded from an sqlite database.